### PR TITLE
#410 inbound detail view

### PR DESCRIPTION
**Pull request by joshxg**
- Starting #410 and I wanted to shift some files around from being in the outbound folder to the top level invoices folder so I could use stuff in the inbound module. So, thought to just PR this so the diff of moving things around goes away!

### CODE REVIEW

File: packages/common/src/hooks/useSortedData/useSortedData.ts [hunk 1]
```diff
@@ -1,33 +1,8 @@
...
-  if (valueA < valueB) {
-    return desc ? 1 : -1;
-  }
-  if (valueA > valueB) {
-    return desc ? -1 : 1;
-  }
-
-  return 0;
-};
-
```
Comment (joshxg) [inline packages/common/src/hooks/useSortedData/useSortedData.ts:30]: I think I have copy pasted this in about 12 places. So I put them all in `common/utils/arrays/sorters`

File: packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx [hunk 2]
```diff
@@ -14,13 +14,13 @@ import {
...
-import { getOutboundShipmentDetailViewApi } from '../../api';
+import { getOutboundShipmentDetailViewApi } from './api';
 import { GeneralTab } from './tabs/GeneralTab';
 import { ItemDetailsModal } from './modals/ItemDetailsModal';
 
-import { OutboundShipmentSummaryItem } from './types';
+import { OutboundShipmentSummaryItem } from '../../types';
 import { Toolbar } from './Toolbar';
-import { isInvoiceEditable } from '../utils';
+import { isInvoiceEditable } from '../../utils';
```
Comment (joshxg) [inline packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx:23]: moved `Invoices/Outbound/utils`  -> `Invoices/utils` to use them in `Invioce/Inbound` also

Comment (joshxg) [reply packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx:23]: Same with types

**Review by mark-prins — APPROVED** [review]
👍  looks good to me!

**Pull request merged**